tutorial-registry/schema.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/scverse/scverse-tutorials/main/schema.json",
+    "title": "Scverse Tutorials",
+    "description": "A tutorial for an scverse core or ecosystem package that is listed on scverse.org/learn",
+    "type": "object",
+    "properties": {
+        "name": {
+            "description": "The name of the tutorial. Must be unique.",
+            "type": "string"
+        },
+        "description": {
+            "description": "Summary of the tutorial (1-2 sentences).",
+            "type": "string"
+        },
+        "link": {
+            "description": "Link to the tutorial",
+            "type": "string",
+            "format": "uri"
+        },
+        "level": {
+            "description": "Difficulty of the tutorial",
```

Comment:
Maybe two categories are enough? Not sure how important this is to begin with. Ideally they're all simple and structured.

Reply:
It's more about where to start first. E.g. last hackathon we discussed you should have understood single-cell processing using `normalize_total` + `log1p` before moving to processing with scvi-tools. 

Also you should have understood what the AnnData slots are for before reading a tutorial how to use distributed dask arrays with AnnData.